### PR TITLE
[0.12] Doc: Wallet & Pruning 

### PR DESCRIPTION
**Pull request by maflcko**
Fixes https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2016-January/012245.html

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -263,6 +263,13 @@ sanity check. Since 0.12, these are no longer stored. When loading a
 0.12 wallet into an older version, it will automatically rescan to avoid
 failed checks.
 
+Wallet: Pruning
+---------------
+
+With 0.12 it is possible to use basic wallet functionality in prunded mode.
```
Comment (jonasschnelli) [inline doc/release-notes.md]: I think we should remove `basic`. Expect of the import and rescan features, you can use everything. `Basic` may imply that only a smaller subset of functions are available.

**Comment by jonasschnelli** [comment]
Nice Thanks!

File: doc/release-notes.md [hunk 2]
```diff
@@ -263,6 +263,13 @@ sanity check. Since 0.12, these are no longer stored. When loading a
 0.12 wallet into an older version, it will automatically rescan to avoid
 failed checks.
 
+Wallet: Pruning
+---------------
+
+With 0.12 it is possible to use wallet functionality in prunded mode.
```
Comment (laanwj) [inline doc/release-notes.md]: s/prunded/pruned

**Comment by laanwj** [comment]
Nice, ACK

Maybe add:
```
To enable block pruning set `prune=N` on the command line or in bitcoin.conf, 
where N is the number of MB to allot for raw block & undo data.
```

**Comment by fanquake** [comment]
ACK

**Comment by MarcoFalke** [comment]
Done. (Note: Pruning is also in `MiB`)

**Pull request merged**